src/Onboarding/ActivateProximityTracing.tsx

```diff
@@ -0,0 +1,104 @@
+import React, { FunctionComponent } from "react"
+import {
+  ScrollView,
+  SafeAreaView,
+  View,
+  StyleSheet,
+  TouchableOpacity,
+} from "react-native"
+import { useTranslation } from "react-i18next"
+import { useNavigation } from "@react-navigation/native"
+
+import { usePermissionsContext } from "../PermissionsContext"
+import { OnboardingScreens } from "../navigation"
+import { GlobalText } from "../components"
+import { Button } from "../components"
+
+import { Spacing, Typography, Buttons, Colors } from "../styles"
+
+const ActivateProximityTracing: FunctionComponent = () => {
+  const { t } = useTranslation()
+  const navigation = useNavigation()
+
+  const { exposureNotifications } = usePermissionsContext()
+
+  const headerText = t("onboarding.proximity_tracing_header")
```

Comment:
Super small comment here but how do you feel about naming the variables closer to the translation keys? It makes the relationship for the values a bit better to read. Maybe just inlining the translations all together?

Reply:
💯